middlewares/apiClient.js

```diff
@@ -2,19 +2,31 @@ const API_ROOT = require('../config.json').apiRoot;
 
 const callApi = (endpoint, method, headers, data) => {
   const fullUrl = (endpoint.indexOf(API_ROOT) === -1) ? API_ROOT + endpoint : endpoint;
-
   return new Promise((resolve, reject) => {
     fetch(fullUrl, {
       method,
-      headers
+      headers,
+      body: JSON.stringify(data)
     })
     .then(response => {
+      console.log(response);
       response.json()
```

Comment:
Only FYI (no changes requested, maybe for a refactor PR): If you return he promise you can chain promises.